### ISSUE TITLE
Raise bower-asset/inputmask requirement to 5.0.9 or greater

### ISSUE THREAD

**Issue by lourdas** [yiisoft/yii2#20265]
Inputmask has version 5.0.9 available, but the composer.json file in the framework specifically requires version 5.0.8. Maybe it's a good idea to change the version from `"^5.0.8"` to `"^5.0"` in composer.json.

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.51
| PHP version      | 7.4.33
| Operating system | Linux

**Comment by schmunk42** [comment]
5.0.9 should get installed, see: https://jubianchi.github.io/semver-check/#/^5.0.8/5.0.9

But there's a space at the end...
https://github.com/yiisoft/yii2/blob/3c75ff1043cdfc3c0c78ad8a4b477b5894223a5a/composer.json#L79

**Comment by craiglondon** [comment]
@rob006 Yeah, sounds like a bug over there.  5.0.9 does exist

```
{
    "name": "bower-asset/inputmask",
    "version": "5.0.9",
    "source": {
        "type": "git",
        "url": "https://github.com/RobinHerbots/Inputmask.git",
        "reference": "310a33557e2944daf86d5946a5e8c82b9118f8f7"
    },
    "dist": {
        "type": "zip",
        "url": "https://api.github.com/repos/RobinHerbots/Inputmask/zipball/310a33557e2944daf86d5946a5e8c82b9118f8f7",
        "reference": "310a33557e2944daf86d5946a5e8c82b9118f8f7"
    },
    "require": {
        "bower-asset/jquery": ">=1.7"
    },
    "type": "bower-asset",
    "license": [
        "http://opensource.org/licenses/mit-license.php"
    ]
},
```

**Comment by schmunk42** [comment]
FYI @rob006 @samdark https://github.com/hiqdev/asset-packagist/issues/169 jQuery 3.7.1 is missing again.

This leads to serious problems, because composer downgrades several packages and it's not obvious to the user.

**Comment by terabytesoftw** [comment]
> [@terabytesoftw](https://github.com/terabytesoftw) we should gradually switch to your foxy fork, I think...

I suggest doing the same solution for the master branch, we add the `package.json`, without any default asset manager, we add the documentation, and the user will be free to use whatever they want, they can use `foxy`, `npm`, `bun`, `pnpm`, `yarn`, i would add foxy to suggestions composer, and to dev, so if the user wants to use it they can.

**Comment by samdark** [comment]
Yeah. Using foxy in app-basic / app-advanced would be more than enough.

**Comment by terabytesoftw** [comment]
> Yeah. Using foxy in app-basic / app-advanced would be more than enough.

So we leave the core without assets.
